Assets/scss/components/quesionnaire.scss

```diff
@@ -0,0 +1,276 @@
+.cmp-questionnaire {
+  max-width: toRem(1175);
+  margin: auto;
+
+  @include max-screen(sm) {
+    max-width: toRem(767);
+    margin: auto;
```

Comment:
This margin and the one in the `between` breakpoint are extra since they will both inherit the higher `cmp-questionnaire` margin that's the same.

Reply:
Thanks for the Feedback this has been addressed have removed the redundant margin properties